index.html

```diff
@@ -94,7 +102,7 @@ <h3 class="event-venu">@Khanapara, Guwhati-36, India</h3>
         <h2>Major Programs</h2>
         <div class="dash"></div>
         <div class="programs">
-          <div class="single-program">
+          <article class="single-program">
```

Comment:
It looks great! but you can improve it by adding the hover effect to the program cards, and aligning the link to the center. So please make sure to fix it.

![image](https://user-images.githubusercontent.com/34779928/197009246-470a5fef-619b-44fd-93f2-6e5dbcd81151.png)

Reply:
Hi @aimalamiri, This is updated in the featured branch. the deployment is of the main branch which may be the reason You are seeing this.

Thanks for pointing out the hover effect though.